SPECS/grub2/grub2.spec

```diff
@@ -335,9 +336,20 @@ cp $GRUB_PXE_MODULE_SOURCE $EFI_BOOT_DIR/$GRUB_PXE_MODULE_NAME
 %{_libdir}/grub/*
 %endif
 
+%files mkconfig
```

Comment:
I am very much against randomly creating new subpackages in the middle of a stable major release. This type of refactoring of subpackages should be done at the Mariner 3.0 boundary

Reply:
The second PR to swap over to grub2-mkconfig will also be done at the 3.0 boundary, correct? I can just leave these changes (subpackage and dependencies) for that PR.

Reply:
Yup, the swap over to a defined subpackage for `grub2-mkconfig` should only be done at the 3.0 boundary. This subpackage change should be reserved for our 3.0 development window

Reply:
Reverted this change.